### ISSUE TITLE
save settings

### ISSUE THREAD

**Issue by RealPetChicken** [BrunoReX/jmkvpropedit#1]
anyway to save the settings, i have to re-put the info everytime the program closes.

**Comment by sebfrie** [comment]
I just want to second that. Saving a preset of settings would be very useful.

**Comment by billybobaloo** [comment]
I agree with the previous posts on this issue.  I use the program to change all my mkv files to language: English and default track: yes.  All of my video files have only 1 video and 1 audio track so the ability to make these choices default would make this program exponentially quicker to use.  Either have the program remember the users last choices or allow the user to save profiles or put in a settings tab.  Excellent program otherwise.

**Comment by SpudR** [comment]
+1 :)

**Comment by BrunoReX** [comment]
A "bit" late but I could add a setting to set the default language for the video/audio/subtitle tracks separately, this would apply to all tracks by default for coding simplicity sake.

**Comment by SpudR** [comment]
Was kinda hoping for something like saving the options set to the .ini file.
I always set the title to be the {file_name}, video and audio track 1 to {file_name}, and delete all .jpg attachments.

**Comment by chippie** [comment]
+1 Would be great if there was an option to seve settings, preferably to the .ini as mentioned.

Perhasp even the option to have multiple 'user' named saves so you can easily load a certain config.

Thanks for all your efforts on this! Been using it since it was released years ago and has been a lifesaver to me! :)

**Comment by chippie** [comment]
Is anything moving on this at all please? Is someone else able to take it over if not?
Really wish I was up to the job to be able to add the option to save the settings.
I am not fully able-bodied and it takes me a good ten minutes to enter the few settings each time. Even though the program stays open, if I have to reboot etc. then it is very time consuming for me unfortunately.
Would really be great if someone was able to add this for us.
Thank You.

**Comment by mrjohnpoz** [comment]
Yeah +1 on this as well.. Even if was something you could load with your settings..  I normally just leave it open so don't have to enter settings every time.. But anytime restart the machine, or on accident close it - have to through and pick all the settings again.

**Comment by mrjohnpoz** [comment]
Still nothing on this?  Couldn't we please add like a .ini file that would load a default config.. This would be such a nice feature, vs having to setup up what I like every time load the program.

**Comment by SpudR** [comment]
Amusingly there's a .ini file that is generated to point at the executable - should be simple to use this file...

**Comment by mrjohnpoz** [comment]
Yeah I am just at a loss to why no traction on this... This request is from 2015..  If it not something developers are willing to do.. Then just close the request with a statement - not going to happen people, or something.. 

I was so excited when I noticed the 1.5.2 update - I was like maybe?? A way to save default settings, or load them, from a file?

I would say like 99/100 files I run through jmkvpropedit always use the same settings.. I use it so often that I just have to leave it open all the time, because running through all the settings every time is just a real pita.  If some off file where different lang or something on audio or sub, or need to set forced track, etc. Those are no big deal..  But I have settings I want on every tab, even set to delete attachments with IDs 1-4, etc.  Maybe its time to look at alternate way of doing what I am doing - but this is so easy to use once the settings are in place.

**Comment by SpudR** [comment]
Pleeeeeeeeeeeeeeease...!!

**Comment by DanTheMobileComputerMan** [comment]
Saving configuration settings is absolutely essential to make this as user friendly as possible, as everyone else here has commented. Should be easy to include, surely. Thanks.

**Comment by DA-iNFAMOUS-DRE** [comment]
Please add a way to save settings

**Comment by ozdeadmeat** [comment]
+1

**Comment by csandman** [comment]
Any updates on this? I love this tool, but have a bunch of different configs I'd like to use over time. For example, I'll often mix tracks from a bunch of different sources for anime together, but for a single show, the resulting tracks would all have the same properties. It would be nice to be able to apply these props to all episodes I have merged multiple times without keeping the program open or setting them all again every time.

In this case, I'm setting a bunch of properties, such as

- A name for every track
- A language for every track
- Whether or not each track is default
- An original language flag for some tracks
- A hearing impaired flag for some tracks

And these are applied to 6 different tracks, making it cumbersome to set them all every time.

Also, it would be nice if I could use one of these groups of settings as a base, because when I do a different show, it would be easier to have a starting point and just change the fields I need (like the name of each track).

**Comment by SebastlAS** [comment]
> Any updates on this? I love this tool, but have a bunch of different configs I'd like to use over time. For example, I'll often mix tracks from a bunch of different sources for anime together, but for a single show, the resulting tracks would all have the same properties. It would be nice to be able to apply these props to all episodes I have merged multiple times without keeping the program open or setting them all again every time.
> 
> In this case, I'm setting a bunch of properties, such as
> 
> * A name for every track
> * A language for every track
> * Whether or not each track is default
> * An original language flag for some tracks
> * A hearing impaired flag for some tracks
> 
> And these are applied to 6 different tracks, making it cumbersome to set them all every time.
> 
> Also, it would be nice if I could use one of these groups of settings as a base, because when I do a different show, it would be easier to have a starting point and just change the fields I need (like the name of each track).

First of all, big  thanks to @BrunoReX for making this usefull tool. i wish tha the tools has the option for saving profiles.
